students/larry_mburu/Lesson02/fizz_buzz.py

```diff
@@ -0,0 +1,14 @@
+def fizz_buzz(): 
+    for num in range(1, 16): 
+        if (num % 3 == 0 ) and (num % 5 == 0):
+            print(num)
+            print("FizzBuzz") 
+        if num % 3 == 0: 
+            print(num)
+            print("Fizz")
+        if num % 5 == 0: 
+            print(num)
+            print("Buzz")
+
+if __name__ == '__main__': 
+    fizz_buzz()
```

Comment:
Close, but not quite.  You're output should be:
1
2
Fizz
4
Buzz
Etc, all the way to 100